docs/HowToConfigureMemory.md

```diff
@@ -1,26 +1,26 @@
 # How to configure Memory
 
 ## Contents
-> [Configuration](#configuration)  
-> [Linker Scripts](#linker-scripts)  
-> [Multi-Heap Support](#multi-heap-support)
+- [Configuration](#configuration)
+- [Linker Scripts](#linker-scripts)
+- [Multi-Heap Support](#multi-heap-support)
 
 ## Configuration
 There are two configurations to configure memory, CONFIG_RAM_REGIONx_START and CONFIG_RAM_REGIONx_SIZE.  
 Menuconfig helps to change them as shown below:
+1. Execute make with the menuconfig option
 ```
-cd $TIZENRT_BASEDIR
-cd os
-make menuconfig
+	cd $TIZENRT_BASEDIR
+	cd os
+	make menuconfig
 ```
-Find configs with following step:
-```
-Hardware Configuration -> Chip selection -> Boot Memory Configuration
-```
-Set CONFIG_RAM_REGIONx_START as hexa value and set CONFIG_RAM_REGIONx_SIZE as decimal value in bytes.
+1. To find configs, select `Hardware Configuration -> Chip selection -> Boot Memory Configuration`
+
+
+1. Set `CONFIG_RAM_REGIONx_START` as hexa value and set `CONFIG_RAM_REGIONx_SIZE` as decimal value in bytes.
 ```
-CONFIG_RAM_REGIONx_START=0x02023800
-CONFIG_RAM_REGIONx_SIZE=968704
+	CONFIG_RAM_REGIONx_START=0x02023800
+	CONFIG_RAM_REGIONx_SIZE=968704
```

Comment:
I don't know what is your policy for spaces. Bash commands? or all of content in box?

Reply:
According to the suggestion, any content within box, especially within numbered lists calls for indentation. However, if the line wraps around to the next, then it is better to let it remain unchanged. Otherwise readability is affected. I noticed the numbering above is wrong, and the box start-end needs indenting too. I will change accordingly, thank you.

Reply:
I have no idea why we make a difference between boxes in numbered and boxes in others.
In your change there are three kinds of boxes
1. no indentation for box and content
2. no indentation for box, indentation for content
3. indentation for box and content
Let's keep the same usages.

Reply:
Thanks for spotting the inconsistency. I am going ahead with option 3 above, in line with the suggestions.